model/team.py

```diff
@@ -55,21 +67,21 @@ def get_platform(self):
         """Return this team's working platform."""
         return self.__platform
 
-    def add_member(self, slack_id):
-        """Add a new member's Slack ID to the team's set of members' IDs."""
-        self.__members.add(slack_id)
+    def add_member(self, gid):
```

Comment:
Inconsistent - it would probably be better if these parameters are also name `guid` (although I don't like the name `guid` either).